.github/workflows/check.yml

```diff
@@ -10,4 +10,9 @@ jobs:
         with:
           ruby-version: '3.2'
           bundler-cache: true # runs 'bundle install' and caches installed gems automatically
-      - run: bundle exec rake typecheck
+      - name: Check for stale signature files
+        run: bundle exec rake rbs:stale
+      - name: Check for missing signature files
+        run: bundle exec rake rbs:missing
```

Comment:
@ivoanjo 

Regarding your message: 

> It would be nice to have something that checks when a file is modified if the rbs counterpart still has untyped and reject the PR until the rbs is typed with the correct types :smile:

> I actually strongly disagree on this, for a simple reason: we're an open-source project that wants community contributions.

We can tweak the check inside the GitHub action using `${{github.actor}}` information. That way we do not bother the community with adding types. Not suggesting doing on this PR 

Regarding the UX of `bundle exec rake rbs:stale`, and  `bundle exec rake rbs:missing` I agree that we could add a nice message `tasks/rbs.rake` explaning what command to run to fix the issue. `bundle exec rake rbs:clean`, and `bundle exec rake rbs:prototype` respectably

Reply:
+1 if this can be gated only to the "usual suspects" I have no concerns at all with the approach and think it's a great idea.

Reply:
Sure, reverted the commit that adds the missing check. I'll do that in a subsequent PR. I kept the stale one because it's much less likely contributors remove files until I add the messages.

Reply:
> I kept the stale one because it's much less likely contributors remove files until I add the messages.

Bah, I reverted it too.

Reply:
Addressed in https://github.com/DataDog/dd-trace-rb/pull/2865